<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="icon_menu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QPushButton" name="icon_menu_toggle_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="icon_menu_today_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="icon_menu_calendar_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="full_menu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>251</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QPushButton" name="full_menu_toggle_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="full_menu_today_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="full_menu_calendar_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="full_menu_tasks_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="workspace_widget" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>711</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QWidget" name="title" native="true">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>30</y>
       <width>641</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QLabel" name="title_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>621</width>
       <height>401</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
